crates/biome_formatter/src/comments.rs

```diff
@@ -1174,35 +1174,37 @@ where
     }
 }
 
-/// Returns `true` if `comment` is a multi line block comment:
+/// Returns `true` if `comment` is a multi line block comment where each line
+/// starts with a star (`*`). These comments can be formatted to always have
+/// the leading stars line up in a column.
 ///
 /// # Examples
 ///
 /// ```rs,ignore
-/// assert!(is_doc_comment(&parse_comment(r#"
+/// assert!(is_alignable_comment(&parse_comment(r#"
 ///     /**
 ///      * Multiline doc comment
 ///      */
 /// "#)));
 ///
-/// assert!(is_doc_comment(&parse_comment(r#"
+/// assert!(is_alignable_comment(&parse_comment(r#"
 ///     /*
 ///      * Single star
 ///      */
 /// "#)));
 ///
 ///
-/// // Non doc-comments
-/// assert!(!is_doc_comment(&parse_comment(r#"/** has no line break */"#)));
+/// // Non indentable-comments
+/// assert!(!is_alignable_comment(&parse_comment(r#"/** has no line break */"#)));
 ///
-/// assert!(!is_doc_comment(&parse_comment(r#"
+/// assert!(!is_alignable_comment(&parse_comment(r#"
 /// /*
 ///  *
 ///  this line doesn't start with a star
 ///  */
 /// "#)));
 /// ```
-pub fn is_doc_comment<L: Language>(comment: &SyntaxTriviaPieceComments<L>) -> bool {
+pub fn is_alignable_comment<L: Language>(comment: &SyntaxTriviaPieceComments<L>) -> bool {
```

Comment:
I renamed this to `is_alignable_comment`, since it's _technically_ not the same as a doc comment when there's only one star, but we can still format the alignment even in that case. This also more closely matches Prettier's `isIndentableBlockComment` naming.

All of the existing usages have also been renamed, of which there were only like 2 anyway.